module/config.go

```diff
@@ -32,6 +34,7 @@ type configProxy struct {
 	Registration xep0077.Config `yaml:"mod_registration"`
 	Version      xep0092.Config `yaml:"mod_version"`
 	Ping         xep0199.Config `yaml:"mod_ping"`
+	Muc          muc.Config     `yaml:"mod_muc"`
```

Comment:
why not `xep0045.Config` like the rest?

Reply:
I guess muc is more descriptive, and since roster and offline are not in the form xepNNNN I thought it would be ok. But fair, I can change it.